tools/snippet_shell.py

```diff
@@ -45,24 +45,24 @@ def _start_services(self, console_env):
         console_env['s'] = self._ad.snippet
 
     def _get_banner(self, serial):
-        return """
-Connected to {}. Call methods against:
-    ad (android_device.AndroidDevice)
-    snippet or s (Snippet)
-    """.format(serial)
-
+        lines = ['Connected to %s.' % serial,
+                 'Call methods against:',
+                 '    ad (android_device.AndroidDevice)',
+                 '    snippet or s (Snippet)']
+        return '\n'.join(lines)
```

Comment:
Yes, this way the code doesn't have to have weird indentation that doesn't follow the regular style just to get around space issues in string; and it's still pretty easy to figure out what the final string looks like.

I changed the breaks a little bit. Can change it back if you prefer the old way